randovania/games/samus_returns/gui/ui_files/preset_msr_hints.ui

```diff
@@ -108,23 +108,23 @@
          </widget>
         </item>
         <item>
-         <widget class="QGroupBox" name="hint_baby_group">
+         <widget class="QGroupBox" name="hint_fbi_group">
           <property name="title">
-           <string>Baby Metroid</string>
+           <string>Final Boss Item</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
-            <widget class="QLabel" name="hint_baby_label">
+            <widget class="QLabel" name="hint_fbi_label">
              <property name="text">
-              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This controls how precise the Baby Metroid hint after collecting all Metroid DNA is.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;No hint&lt;/span&gt;: No useful information is provided.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Show only the area name&lt;/span&gt;: The hint reveals the area in which the Baby Metroid is located (&amp;quot;Area 2&amp;quot;, &amp;quot;Area 7&amp;quot;, etc.)&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Show area and room name&lt;/span&gt;: Same as above, but shows the exact room name.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This controls how precise the Final Boss Item hint after collecting all Metroid DNA is.&lt;/p&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Diggernaut needs Bomb&lt;/li&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Metroid Queen needs Ice Beam&lt;/li&gt;&lt;/ul&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Proteus Ridley needs the Baby Metroid&lt;/li&gt;&lt;/ul&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;No hint&lt;/span&gt;: No useful information is provided.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Show only the area name&lt;/span&gt;: The hint reveals the area in which the Baby Metroid is located (&amp;quot;Area 2&amp;quot;, &amp;quot;Area 7&amp;quot;, etc.)&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Show area and room name&lt;/span&gt;: Same as above, but shows the exact room name.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
This doesnt feel clear enough for me that its referring with the bosses being final. Also I would definitely have proteus ridley at the top because that's default

Reply:
![image](https://github.com/user-attachments/assets/a595f348-3122-4f98-9396-beac6fd9849c)